### PR TITLE
gsoc: Make "Chain Actions" default

### PR DESCRIPTION
**Pull request by Nosferatul**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
shouldnt do nothing be default? am i missing something>

**Comment by Nosferatul** [comment]
@adrianzatreanu i'll add a screenshot. By default you can just choose a single action, with this, you can choose multiple actions like when we use chain action

**Comment by Nosferatul** [comment]
http://imgur.com/bmyJVi3

**Comment by yukiisbored** [comment]
euhm, I think the commit message is a bit confusing...

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -641,50 +641,42 @@ def choose_action(console_printer, actions, apply_single=False):
...
-    choice = choose_action(console_printer, actions, apply_single)
 
-    if choice == 0:
-        return None, None
+            for c in choice:
+                c = str(c)
+                actions_desc_len = len(actions_desc)
+                if c.isnumeric() or c == '0':
+                    if c == '0':
+                        actions_desc.append('Do (N)othing')
```
Comment (yukiisbored) [inline coalib/output/ConsoleInteraction.py]: Is there another way to do this? This looks like a hack :/

Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: Well, I need somehow to check for `Do Nothing` action. I think it was a hack already there

Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: ugh this code is horrible

Comment (Nosferatul) [inline coalib/output/ConsoleInteraction.py]: I don't know if we will need this if we implement https://github.com/coala/coala/pull/4629

**Comment by Adrianzatreanu** [comment]
looks decent to me, @yukiisbored ?

**Comment by yukiisbored** [comment]
@Adrianzatreanu It looks alright, it's a fix from the old code. Do we need review from other people?

**Comment by Adrianzatreanu** [comment]
ack d0589f3

**Comment by Nosferatul** [comment]
reack 0911078

**Comment by Nosferatul** [comment]
@rultor merge

**Pull request merged**